### PR TITLE
News update for 1.7.0-rc1

### PR DESCRIPTION
**Pull request by shamisp**
Signed-off-by: Pavel Shamis (Pasha) <pasharesearch@gmail.com>

### CODE REVIEW

**Comment by shamisp** [comment]
@tonycurtis please take a look

**Comment by shamisp** [comment]
@yosefe please take a look.

**Comment by tonycurtis** [comment]
NEWS:

14: - Added support multiple listening transports

support for…

17: - Added a API to retrieve the server listening port

“an API”, or just drop the “a” to be consistent with other entries

19: - Remove deprecated API for querying UCT memory domains

“Removed” to be tense consistent

38: 
- Add Bull Atos HCA device IDs
39: - Add Azure Pipelines testing

“Added” to be tense consistent

Tony

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12816/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 9 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 02-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/231fa7faef7e05eaed8feaa50a3358b199348bac/_main) | :x: FAILURE
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/6be3b1612becf49c9c32029b82c40720c6a1f7c8/hpc-arm-hwi-jenkins_W0) | :x: FAILURE
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/15fdc388f3692aacd0793d461771963d91c546d0/hpc-arm-hwi-jenkins_W1) | :x: FAILURE
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/3b5904c5d812d28bea9c5537298ee5d4bc5aa710/hpc-arm-hwi-jenkins_W2) | :x: FAILURE
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/d6cf9d5fcd1bbf309626069b8fc49bcd23351883/hpc-arm-hwi-jenkins_W3) | :x: FAILURE
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/db8ffc562ee89035a17aac37fc225362cb894aae/hpc-test-node-gpu_W1) | :x: FAILURE
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/c90ce16a86ee617f379e550e19388308b47f12b0/hpc-test-node-gpu_W2) | :x: FAILURE
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/bb54f6f1908531c85f501375af73a1d67873c311/hpc-test-node-gpu_W3) | :x: FAILURE
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/c631094b3832732e019b9c7daa622ee626d1e27e/hpc-test-node-legacy_W0) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/6b6f4239d65c611753001c8eed28024084620b63/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/583397a5766f8a0d50b5ba690e3a8054085a1922/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/aa0e8aa96242506d73b39aacadd04fdc42ba7fb5/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/2b671fbb7159441c0d151b895dc35e71ab12ef90/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/4860af0c362140d44a964ebc1b00ec59fdad9567/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/43add988fd3d4b604dacf7ed527ce8dbac07d4b1/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/5a5ac12f54d772af40c55f02514f004eb2d905ae/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/9ce7ba375554d999640f30a9ed01336574f0a7b3/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/945ee5de3d93014bc9da57559284cb5f8a5235ff/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/8d813fda50ec695b65bfc85e42bb3da342938020/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/48cca60c5338cbc463cf2989315e047d1a1e427b/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/2ae306a52e9fde2ee25233f15561a0bba297c4d6/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/0af9bac605898150371356ef58be63d19a94bd3d/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/059dd7720933d60b0b134e3f7b755aa2b381d633/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/922e44383197de0ffaf493e319516170d403baff/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/359920504c8b1b5623823bdefab46e1edba67128/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/2260941b08dc37196df0a8fca760d628d6bd1c9f/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/973bf7bc2a9db29cc8117fc43405cc7831e106aa/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/3d88a7dbce6dc93b96c07f0f8b726d9e98a3cbe4/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/92a7092c924fac4aa615be91f21dd05b/raw/0ec55520219f06d36575d2006ff2ecf63f365ebd/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

File: NEWS [hunk 1]
```diff
@@ -1,15 +1,48 @@
...
+## 1.7.0 (October 26, 2019)
 Features:
-- TBD
+- Added support multiple listening transports
+- Added UCT socket based connection manager transport
+- Updated API for UCT component management
+- Added a API to retrieve the server listening port
+- Added UCP active message API
+- Remove deprecated API for querying UCT memory domains
+- Added new UCT component query API and deprecate old one
```
Comment (yosefe) [inline NEWS]: duplication of line 16?

File: NEWS [hunk 2]
```diff
@@ -1,15 +1,48 @@
...
+- Refactored server/client examples
+- Added support for dlopen interception in UCM
+- Added non-blocking TCP/IP connection manager
+- Added socket based connection manager
```
Comment (yosefe) [inline NEWS]: duplication of line 15?

File: NEWS [hunk 3]
```diff
@@ -1,15 +1,48 @@
...
+- Added support for PCIe atomics
+- Updated Java API
+- Updated support for Mellanox DEVX API
```
Comment (yosefe) [inline NEWS]: Updated support for Mellanox DevX API

File: NEWS [hunk 4]
```diff
@@ -1,15 +1,48 @@
...
+
```
Comment (yosefe) [inline NEWS]: to add:
- TCP transport performance optimizations
- Improved memcpy() for Intel platforms

File: NEWS [hunk 5]
```diff
@@ -1,15 +1,48 @@
...
+## Copyright (C) UT-Battelle, LLC. 2014-2019. ALL RIGHTS RESERVED.
+## Copyright (C) ARM Ltd. 2017-2019.  ALL RIGHTS RESERVED.
 ##
 ## See file LICENSE for terms.
 ##
 #
 
 ## Current
+
+## 1.7.0 (October 26, 2019)
```
Comment (yosefe) [inline NEWS]: pls replace 1.7.0 by 1.7.0-rc1

File: NEWS [hunk 6]
```diff
@@ -1,15 +1,48 @@
...
+- Added a API to retrieve the server listening port
+- Added UCP active message API
+- Remove deprecated API for querying UCT memory domains
+- Added new UCT component query API and deprecate old one
+- Refactored server/client examples
+- Added support for dlopen interception in UCM
+- Added non-blocking TCP/IP connection manager
+- Added socket based connection manager
+- Added support for PCIe atomics
+- Updated Java API
```
Comment (shamisp) [inline NEWS]: @petro-rudenko - can you help me be more specific about Java API updates ?

Comment (petro-rudenko) [reply NEWS]: Java API, supports most of UCP layer operations (context init, memory registration, creating workers, endpoints, listeners, non blocking tag-send/tag-recv/get/put, enpoint and worker flush).

**Comment by shamisp** [comment]
@yosefe @yosefe @petro-rudenko thanks for review comments. I updated news file.

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12857/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>FAILED</b> on 5 of 29 workers (click for details)</summary>

**Note:** the logs will be deleted after 04-Nov-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/5199525e461b7125d92e34726cb68cf8744d6367/_main) | :x: FAILURE
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/68dc4a73c40daee97319e7ea90a475a386d063e3/r-vmb-ppc-jenkins_W0) | :x: FAILURE
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/c12853b10540202ef188773bc0aa3e4ee6b3b549/r-vmb-ppc-jenkins_W1) | :x: FAILURE
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/6eaf66c16aff0a850161b0da88dc700f15cf5061/r-vmb-ppc-jenkins_W2) | :x: FAILURE
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/f9a82318bcb5393913ffd7e6caaa768140005430/r-vmb-ppc-jenkins_W3) | :x: FAILURE
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/836a3e8e5d5def46e8b996858ed37f31d369ca78/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/6ca11dcb6f9dad149e0b0ac0ed22168dd5436c37/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/49e46ecdce7e3bcbc60e468c5d7ed8c1b920aff5/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/14c3c52f199bcbca7e814c86b990ad889f0c59d9/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/2c06ada0cddffafe33d5919edfc9273be2186d65/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/2446aed92bca4db097950079f677df6b6a7ef582/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/f3d6125fc18d4e6d5ceecbde60e600d007c4b306/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/60fae48723bd11b64c99f5c79e866324fc3c3007/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/7ad99903786dd3f7443b5f1201e864440a610a12/hpc-test-althca_W0) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/3f94edb1306389eef30275eb9ad8e0dbb459ce58/hpc-test-althca_W1) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/ddfd6ec703baa5fed461dd6571072e4f36b4f61c/hpc-test-althca_W2) | :heavy_check_mark: SUCCESS
[hpc-test-althca_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/3f63fcf2030560c4265cff5ee273287e3f35a635/hpc-test-althca_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/44a81a5ca2ac5d8270b4ecb902357cbbd49177f5/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/5c017c12395113f380e9142c058a5d7cf8224235/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/08ed2d04f98b8c6ae72132bd43bf90f626e88e06/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/24f6326fb6d4f78be1f4e3fb41ce457ac90ef2be/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/9de8a62a5568aa608a076db81d358c8bb0f98976/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/fcb0a229180c9b32feecf1b4d3cfae211842bde5/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/6889e9c6730aaf659f0683a3d4b6a99863020589/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/df00be3598bf8d0674a00f8460ab509790bef57b/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/eaab967086edc8730bb27e216038c672a72fd04f/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/af43cf96c9a7db65c8390f1f04a04f934f01f074/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/5792dc41569b9d44397cdc1788ef662030f9ca46/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/4cbc54b2d4efe75d8dfa46989e511184/raw/aaa6e47dd27b91ca617d661f6854c40c539075f7/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS

**Pull request merged**